dependenciesWhitelist.txt

```diff
@@ -14,6 +14,8 @@
 @hapi/wreck
 @jest/environment
 @jest/fake-timers
+@jest/jest-diff
+@jest/pretty-format
```

Comment:
These aren't on npm, but `jest-diff` and `pretty-format` are. Is there some reason these are not published in the `@jest` namespace?

* https://www.npmjs.com/package/@jest/jest-diff - not found
* https://www.npmjs.com/package/@jest/pretty-format - not found
* https://www.npmjs.com/package/jest-diff - found (and its github repo points to the jest monorepo)
* https://www.npmjs.com/package/pretty-format - found (and its github repo points to the jest monorepo)

Reply:
@sandersn I have no idea since I don't work on the jest codebase. I'm trying to submit changes over on DefinitelyTyped (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44033) to update typings for `jest-environment-puppeteer` and that [CI process is failing](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44033#issuecomment-616241339) because it can't resolve the `jest-diff` and `pretty-format` modules -- which aren't explicitly referenced in the package I'm trying to update, which is more baffling.

I'm trying to figure out what configuration is needed to get the DefinitelyTyped build process to succeed (I've left many comments over there on the PR with my research and attempts so far).

My latest research is documented at https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44033#issuecomment-617504889 if you have any ideas what the DefinitelyTyped error messages mean.

Thanks